adcm_aio_client/core/objects/cm.py

```diff
@@ -164,11 +164,9 @@ def cluster(self: Self) -> Cluster:
         return self.service.cluster
 
     @cached_property
-    def hosts(self: Self) -> "HostsInClusterNode":
-        return HostsInClusterNode(  # TODO: new ComponentHostsNode
-            path=(*self.cluster.get_own_path(), "hosts"),
-            requester=self._requester,
-            # filter=Filter({"componentId": self.id}),
+    def hosts(self: Self) -> "HostsNode":
+        return HostsNode(
```

Comment:
probably common hosts node should exist that doesn't have create method

Reply:
refactored